api/manifest/manifest_test.go

```diff
@@ -177,6 +177,37 @@ worker:
 	_assert(t, cases)
 }
 
+func TestGenerateDockerfileProcfile(t *testing.T) {
+	destDir := mkBuildDir(t, "../../examples/dockerfile-procfile")
+	defer os.RemoveAll(destDir)
+
+	Init(destDir)
+	m, _ := Read(destDir, defaultManifestFile)
+
+	Execer = func(bin string, args ...string) *exec.Cmd {
+		return exec.Command("true")
+	}
+
+	cases := Cases{
```

Comment:
Hmm. Looking at the comments from the previous PR and the code I don't see where this ever advertised it was generating Dockerfile. It's generating docker-compose.yml when a Dockerfile and a Procfile is present.